### PR TITLE
Get endpoint for messages has been modifed with query parameter

### PR DESCRIPTION
**Pull request by rana3399**

### CODE REVIEW

File: server/api.js [hunk 1]
```diff
@@ -75,24 +75,28 @@ const api = () => {
...
 
-    const getNewMessage = async (req, res) => {
-    try {  
-      const userIdForMassage = parseInt(req.params.userId);
-      const result = await pool.query(`SELECT m.content, times_stamp FROM messages m WHERE m.user_id=$1`, 
-      [userIdForMassage]
-      );
+const getNewMessage = async (req, res) => {
+  try {  
+    const massagesByEventId = req.query.event;
```
Comment (dnaranjo89) [inline server/api.js]: Can we rename `massagesByEventId` to something like `eventId`

File: server/api.js [hunk 2]
```diff
@@ -75,24 +75,28 @@ const api = () => {
...
-      const lastMessage =  mesggageArray[mesggageArray.length - 1];
+    if(resultArr.length === 0){
       res.status(201)
-          .send(lastMessage);
-
-    } catch (err) {
-      console.log(err);
+        .send("No messages yet. Be the first one to start a conversation");
+    }else{
+      res.status(201)
```
Comment (dnaranjo89) [inline server/api.js]:
```suggestion
      res.status(200)
```

File: server/api.js [hunk 3]
```diff
@@ -75,24 +75,28 @@ const api = () => {
...
+  try {  
+    const massagesByEventId = req.query.event;
+    const result = await pool.query(`SELECT * FROM messages m WHERE m.event_id=$1`, 
+    [massagesByEventId]
+    );
+    const resultArr = result.rows;
 
-      const mesggageArray = result.rows;
-      const lastMessage =  mesggageArray[mesggageArray.length - 1];
+    if(resultArr.length === 0){
```
Comment (dnaranjo89) [inline server/api.js]: Let's not consider this case (`resultArr.length === 0`) and instead always return `resultArr`
```
res.send(resultArr)
```

File: server/api.js [hunk 4]
```diff
@@ -75,24 +75,28 @@ const api = () => {
...
-///////////////getNewMessage ////////////
+//----getNewMessage----//
 
-    const getNewMessage = async (req, res) => {
-    try {  
-      const userIdForMassage = parseInt(req.params.userId);
-      const result = await pool.query(`SELECT m.content, times_stamp FROM messages m WHERE m.user_id=$1`, 
-      [userIdForMassage]
-      );
+const getNewMessage = async (req, res) => {
```
Comment (dnaranjo89) [inline server/api.js]: Rename `getNewMessage` to `getMessagesByEventId`

**Comment by rana3399** [comment]
Modified by following David request.

**Pull request merged**